### ISSUE TITLE
Is this an unnecessary duplicate?

### ISSUE THREAD

**Issue by TPS** [svn2github/subversion#1]
@svn2github @Gabrys This mirror seems to be an unnecessary duplicate of https://github.com/apache/subversion.  Maybe you want to delete it to [preclude a possible polite quota e-mail](https://help.github.com/articles/what-is-my-disk-quota/#file-and-repository-size-limitations)?

**Comment by gabrys** [comment]
Actually, the whole svn2github is now outdated, because GitHub now allows you do to the same thing natively

**Comment by TPS** [comment]
I thought so, too, but @GitHub says, no, they only do static importing.…  Sorry for the misinfo!

**Comment by gabrys** [comment]
Oh, do you mean they don't keep the repos updated with the SVN repos?

**Comment by TPS** [comment]
Correct.  Look at the problem reflected in https://github.com/javiergutierrezchamorro/nikkhokkho/issues/1:

https://github.com/javiergutierrezchamorro/nikkhokkho/commits/master vs https://github.com/svn2github/nikkhokkho/commits/master
